pages/home.tsx

```diff
@@ -17,10 +104,18 @@ const Home = ({ user }: ServerPageProps) => {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <Page user={user}>
+        {showQrCode && <QRCodeModal qrCode={qrCode} onClose={onClose} />}
         <Box pad="medium">
-          <Text>Home</Text>
-          <Text>{user.email}</Text>
-          <Text>{user.role}</Text>
+          {user.role === 'patient' ? (
+            <PatientPrescriptions
+              prescriptions={prescriptions}
+              previousPrescriptions={previousPrescriptions}
+              setQrCode={setQrCode}
+              setShowQrCode={setShowQrCode}
+            />
+          ) : (
+            <PharmacyLocationView pharmacyLocation={pharmacyLocation} />
```

Comment:
we have `enums for these` defined in `/types/types.ts` ! 
```suggestion
          {user.role === Role.Patient && (
            <PatientPrescriptions
              prescriptions={prescriptions}
              previousPrescriptions={previousPrescriptions}
              setQrCode={setQrCode}
              setShowQrCode={setShowQrCode}
            />}
            {role === Role.Staff &&<PharmacyLocationView pharmacyLocation={pharmacyLocation} /> }
```
removing the ternary operator makes it a little safer since this way we will only load the components if there is an exact match. i.e lets say the user's role is `undefined` for some reason (maybe a failed fetch or a data error), then in this case we would render the Staff view which we wouldn't want to happen. Instead we would want to show nothing, or throw up an error screen.